### DISCUSSION TITLE
Reader parser not return error

### DISCUSSION THREAD

**Discussion by boaz30333** [open-source-parsers/jsoncpp#1476] [Q&A]
Hi all, 
When I try to parse an invalid string like:
"a":"b"
"a""b,
 "a"..."b" , 
"a";"b"

It returns an answer "a" instead of an error.

Does anyone have any idea how to get to know if the reader skipped some of the characters?
Thanks in advance!

**Comment by BillyDonahue** [comment]
it matters exactly how this data was represented. Give a code example if you can.

**Reply by boaz30333** [reply]
```c++
Json::Value root;
Json::Reader reader;
std::string jsonAsString = "\"a\"\"b\"";
bool isSuucess = reader.parse(jsonAsString, root);
cout << root.toStyledString();
```

isSuccess = true
but  root.toStyledString() = "a"

**Reply by BillyDonahue** [reply]
If I understand, you have as input a raw string that looks like this character sequence:

    "a""b"

And that is supposed to fail JSON parsing but doesn't.
I think I understand. That would be a bug.

**Comment by boaz30333** [comment]
```
Json::Value root;
Json::Reader reader;
std::string jsonAsString = "\"a\"\"b\"";
bool isSuucess = reader.parse(jsonAsString, root);
cout << root.toStyledString();
```

isSuccess = true
but  root.toStyledString() = "a"
